{"name": "test_demo", "status": "failed", "statusDetails": {"message": "AssertionError: 这次真的断言失败了，可能是因为元素的定位问题\nassert 'yuz' not in 'yuz wang'\n  'yuz' is contained here:\n    yuz wang\n  ? +++", "trace": "self = <test_demo.TestDemo object at 0x000001DBCA628D60>\n\n    @pytest.mark.linux\n    @pytest.mark.success\n    def test_demo(self):\n        expected = \"yuz\"\n        acutal = \"yuz wang\"\n>       assert expected not in acutal, \"这次真的断言失败了，可能是因为元素的定位问题\"\nE       AssertionError: 这次真的断言失败了，可能是因为元素的定位问题\nE       assert 'yuz' not in 'yuz wang'\nE         'yuz' is contained here:\nE           yuz wang\nE         ? +++\n\ntest_demo.py:16: AssertionError"}, "start": 1592229559797, "stop": 1592229559797, "uuid": "4a8c9bd7-e537-4cda-8f18-c9b930bf6fcc", "historyId": "da1404464d9f0eb602fb3f161c53aad7", "testCaseId": "95f8837b3a77b7d37f64bf981f432030", "fullName": "test_demo.TestDemo#test_demo", "labels": [{"name": "tag", "value": "demo"}, {"name": "tag", "value": "linux"}, {"name": "tag", "value": "success"}, {"name": "suite", "value": "test_demo"}, {"name": "subSuite", "value": "TestDemo"}, {"name": "host", "value": "LAPTOP-LENSEH1Q"}, {"name": "thread", "value": "17096-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_demo"}]}